subprojects/docs/src/docs/release/notes.md

```diff
@@ -30,6 +30,10 @@ in the next major Gradle version (Gradle 5.0). See the User guide section on the
 
 The following are the newly deprecated items in this Gradle release. If you have concerns about a deprecation, please raise it via the [Gradle Forums](https://discuss.gradle.org).
 
+### Deprecation of `-a`
```

Comment:
:o: I would just using wording here instead of the concrete command line option. Maybe "Deprecation of no-rebuild command line options".

Reply:
Phew...